microsetta_admin/tests/test_metadata_transforms.py

```diff
@@ -0,0 +1,164 @@
+import unittest
+import pandas as pd
+import pandas.testing as pdt
+from microsetta_admin.metadata_transforms import (
+    apply_category_specific_transforms,
+    AgeYears, AgeCat, BMI, BMICat, AlcoholConsumption,
+    NormalizeHeight, NormalizeWeight)
+from microsetta_admin.metadata_constants import MISSING_VALUE
+
+
+class TransformTests(unittest.TestCase):
+    def test_apply_category_specific_transforms(self):
+        df = pd.DataFrame([['1990', 'July', '2020-01-01T12:00:00'],
+                           ['1995', MISSING_VALUE, '2019-01-23T12:00:00'],
+                           [MISSING_VALUE, 'May', '2018-01-23T12:00:00'],
+                           [MISSING_VALUE, MISSING_VALUE,
+                            '2017-01-23T12:00:00']],
+                          index=list('abcd'),
+                          columns=['birth_year', 'birth_month',
+                                   'collection_timestamp'])
+        transforms = (AgeYears, AgeCat)
```

Comment:
Do we have a test that shows what happens when the input dataframe does NOT contain one of the columns required by one of the transforms?  Would like to see that ...

Reply:
Good call, will add